code/modules/xenomorph/xeno_structures.dm

```diff
@@ -57,8 +57,9 @@
 	if(!istype(center_turf))
 		center_turf = loc
 
-	for(var/i in RANGE_TURFS(XENO_SILO_DETECTION_RANGE, src))
-		RegisterSignal(i, COMSIG_ATOM_ENTERED, .proc/resin_silo_proxy_alert)
+	if(SSticker.mode?.flags_round_type & MODE_SILO_RESPAWN)
+		for(var/turfs in RANGE_TURFS(XENO_SILO_DETECTION_RANGE, src))
```

Comment:
maybe put in an AS ? since you're only getting turfs

Reply:
i don't need them as turfs